AGENTS.md

```
@@ -5,79 +5,76 @@ This file provides guidance to AI coding agents when working with code in this r
+- `pnpm clean` - Clean root node_modules
+- `pnpm clean:workspaces` - Run configured workspace clean tasks
+- `pnpm clean:browser-data-isolated` - Remove isolated browser development profiles only
 
 ## Architecture Overview
 
-Stagewise is a monorepo using pnpm workspaces and Turborepo. The product is a browser-based development tool that connects frontend UI to AI agents in code editors, allowing developers to select DOM elements and have AI agents implement changes.
+Stagewise is a pnpm workspace and Turborepo monorepo. The product is an open-source agentic IDE with a built-in coding agent. Its main Electron browser combines browsing, debugging, editing, and agent workflows.
 
 ### Directory Structure
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)